.github/workflows/release.yml

```diff
@@ -15,9 +15,9 @@ jobs:
     environment: production
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
This is the latest major version of this action at the time of this pull request's creation.